frontend/src/metabase/dashboard/analytics.ts

```diff
@@ -0,0 +1,8 @@
+import { trackSchemaEvent } from "metabase/lib/analytics";
+
+export const trackAutoApplyFiltersDisabled = (dashboardId: number) => {
```

Comment:
`DashboardId` type from `metabase-types/api`

Reply:
Oh I missed this comment, I will push the fix directly to the integration branch.

Reply:
Fixed in https://github.com/metabase/metabase/pull/31020/files#diff-7add21813ab25b9039aa3a79f7057e247562a152097d18fa9e6996a1f876066bR4